### PR TITLE
workaround for upstream issue

### PR DESCRIPTION
**Pull request by baentsch**
Working around https://github.com/openssl/openssl/issues/15569

### CODE REVIEW

File: openssl3/fulltest/testrun.py [hunk 1]
```diff
@@ -10,22 +10,28 @@
    jsoncontents = f.read();
 
 assignments = json.loads(jsoncontents)
+ok=True
 for sig in assignments:
     print("Testing %s:" % (sig))
     for kem in assignments[sig]:
-     if not "_" in kem: # hybrids not yet supported in OSSL3-oqsprovider: TBD
+    # if not "_" in kem: # hybrids not yet supported in OSSL3-oqsprovider: TBD
```
Comment (dstebila) [inline openssl3/fulltest/testrun.py:17]: Will this code actually run after line 17 is commented out? Indentation is different once this is commented out, not sure if that's still valid Python.

**Review by dstebila — CHANGES_REQUESTED** [review]

File: openssl3/fulltest/testrun.py (hunk 1 above)
Comment (baentsch) [reply openssl3/fulltest/testrun.py:17]: I'd say so -- as long as nobody moves the "if" statement to the same level as the "for".

Comment (dstebila) [reply openssl3/fulltest/testrun.py:17]: More specifically will the code underneath it still run now that it's at the wrong indentation level?  If you executed the script already then my question can be ignored.

Comment (baentsch) [reply openssl3/fulltest/testrun.py:17]: Yes, absolutely. You can give it a try with `./testrun.sh openquantumsafe/oqs-ossl3`

Comment (baentsch) [reply openssl3/fulltest/testrun.py:17]: But actually, I didn't really mean to include this change in the PR :-( All I wanted to change is the ossl3 Dockerfile. So this is an error. PR will be updated, eliminating this.

**Comment by baentsch** [comment]
Replaced by #85

**Pull request closed without merging**